ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/LocalMediaPlayer.kt

```diff
@@ -167,7 +171,7 @@ class LocalMediaPlayer(
             val mainHandler = Handler(context.mainLooper)
 
             val myRunnable = Runnable {
-                onPlayerStateChanged!!(playerState, currentPlaying)
+                onPlayerStateChanged?.invoke(playerState, currentPlaying)
```

Comment:
This was an old bug. onPlayerStateChanged can be null here, because it may be gone in the time the execution arrives here.